src/v/http/client.cc

```diff
@@ -56,10 +57,16 @@ client::make_request(client::request_header&& header) {
         return ss::make_ready_future<request_response_t>(
           std::make_tuple(req, res));
     }
-    return connect().then([req, res] {
-        return ss::make_ready_future<request_response_t>(
-          std::make_tuple(req, res));
-    });
+    return connect()
+      .then([req, res] {
+          return ss::make_ready_future<request_response_t>(
+            std::make_tuple(req, res));
+      })
+      .handle_exception_type([this](const ss::tls::verification_error& err) {
```

Comment:
if we can't connec to GCP... should we log that we couldn't connect ?

Reply:
The error will be propagated and logged by archival. Otherwise we will log
the same error on multiple levels.

Reply:
got it.